appinventor/appengine/build.xml

```diff
@@ -379,6 +379,7 @@
         <pathelement location="${lib.dir}/guava/guava-18.0.jar" />
         <pathelement location="${lib.dir}/gwt_dragdrop/gwt-dnd-3.2.3.jar" />
         <pathelement location="${lib.dir}/gwt_incubator/gwt-incubator-20101117-r1766.jar" />
+        <pathelement location="${lib.dir}/moxieapps/org.moxieapps.gwt.uploader-1.1.0.jar" />
```

Comment:
I've found other GWT libraries (e.g. https://github.com/fredsa/gwt-dnd) but I used the Moxie uploader because it had clearer documentation and seemed to have more capability for future work. At this point, I'm starting my usability tests today so I can't switch this out. Would it be okay to test the impact of drag and drop to see if it is worth implementing with another library or from scratch?